lib/mobility-core/src/Kernel/Beam/Lib/UtilsTH.hs

```diff
@@ -13,6 +13,7 @@ module Kernel.Beam.Lib.UtilsTH
     mkBeamInstancesForEnumAndList,
     mkBeamInstancesForJSON,
     mkCustomMappings,
+    module Rexport,
```

Comment:
no need to reexport, instances reexported automatically when you added import `import Kernel.Beam.Lib.SqlObjectInstances ()`